loops.js

```diff
@@ -50,4 +65,12 @@
 // >> 8
 // >> 9
 // ==========================================
-
+for (let i = 0; i < 10; i++) {
+    if (i > 2 && i < 6) {
+        console.log(">" + i);
+    }else if (i > 5) {
+        console.log(">>" + i);
+    } else {
+        console.log(i);
+    }
+}
```

Comment:
Heel goed, leuk dat jij het anders had dan ik. Wellicht zijn de else en else-if statements hier overbodig (ook al is het resultaat goed)
Ik weet niet of er technische (compileer tijd etc..) voordelen zijn om alleen if statements te doen.